packages/modals/package.json

```diff
@@ -27,6 +27,7 @@
     "@zendeskgarden/container-modal": "^0.8.0",
     "@zendeskgarden/container-utilities": "^0.5.1",
     "dom-helpers": "^5.1.0",
+    "react-merge-refs": "^1.1.0",
```

Comment:
Nice. Let's definitely capture this as a TODO for exploring the `useCombinedRefs` replacement.